pkg/network/go/goid/internal/generate_goid_lut.go

```diff
@@ -3,8 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2022-present Datadog, Inc.
 
-//go:build ignore
```

Comment:
Did you removed the build ignore on purpose?

Reply:
Good catch – I forgot to mention this. It seems like Go has changed the format of these directives over time and older Go versions (<1.16 if I'm not mistaken) complain about this because they expect `//+build ignore` instead. Given that this program isn't built anyway in our CI pipeline, I thought it was OK to simply get rid of it.

Reply:
I see, I'm fine with removing it as well, as you said it should not be built anyway.

However, let's mention this change in the PR description. If we need it for future reference, this information may be useful.

Reply:
Done!